### PR TITLE
subscription quick fix

### PR DESCRIPTION
**Pull request by dw511214992**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by navysingla — APPROVED** [review]

**Comment by navysingla** [comment]
@qianwens Can we merge this PR?

**Comment by dw511214992** [comment]
Hi @navysingla the configuration of readme.go.md is added for latest tag. Please review it.

**Review by navysingla — APPROVED** [review]

**Comment by navysingla** [comment]
@qianwens Can we merge this PR?

**Comment by akning-ms** [comment]
Hi @navysingla, looks your PR include 2 validation issue:
1.  PrettierCheck failure which need to fix(refer to https://aka.ms/ci-fix)
2. For breaking change which caused by change operation id. Looks 2020-09-01 is a very new version, can we let us whether it is official GA? if yes. you may refer to below process https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/43942/Policy-for-Breaking-Changes
3. Python SDK breaking change (caused by operation id change) Is this expected?

**Comment by dw511214992** [comment]
Hi @akning-ms 
1. PrettierCheck failure which need to fix(refer to https://aka.ms/ci-fix)
Answer: I have fixed PrettierCheck failure.
2. For breaking change which caused by change operation id. Looks 2020-09-01 is a very new version, can we let us whether it is official GA? if yes. you may refer to below process https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/43942/Policy-for-Breaking-Changes
Answer: I change the operation id because the operation id is not in a standard format, which is not friendly to codegen. @navysingla Please check with your team wether we need to refer to the above process.
3. Python SDK breaking change (caused by operation id change) Is this expected?
Answer: I think we may not need to re-release python sdk immediately, but @navysingla please check whether it is allowed.

**Comment by navysingla** [comment]
> Hi @akning-ms
> 
> 1. PrettierCheck failure which need to fix(refer to https://aka.ms/ci-fix)
>    Answer: I have fixed PrettierCheck failure.
> 2. For breaking change which caused by change operation id. Looks 2020-09-01 is a very new version, can we let us whether it is official GA? if yes. you may refer to below process https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/43942/Policy-for-Breaking-Changes
>    Answer: I change the operation id because the operation id is not in a standard format, which is not friendly to codegen. @navysingla Please check wether we need to refer to the above process.
> 3. Python SDK breaking change (caused by operation id change) Is this expected?
>    Answer: @navysingla please check whether it is allowed.

This is newly added version which is added last week. No SDK has been released yet for this version and API is not live yet for real customers, only internal testing users are using it.

File: specification/subscription/resource-manager/readme.go.md [hunk 1]
```diff
@@ -13,11 +13,21 @@ go:
...
   - tag: package-2017-11-preview
 ```
 
+### Tag: package-2020-09 and go
+
+These settings apply only when `--tag=package-2020-09 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2020-09' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/package-2020-09/$(namespace)
```
Comment (ArcturusZhang) [inline specification/subscription/resource-manager/readme.go.md:28]:
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2020-09-01/$(namespace)
```

**Pull request merged**